extensions/wikia/Piggyback/Piggyback_body.php

```diff
@@ -43,10 +49,27 @@ function execute( $par ) {
 			$LoginForm->setDefaultTargetValue( $wgRequest->getVal( 'target' ) );
 		}
 
-		$this->logger->info( 'IRIS-4219 Piggyback has been rendered' );
 
 		$LoginForm->render();
 	}
+
+	private function redirectToMercuryPiggyback( OutputPage $out, $target ) {
+		$redirectUrl = '/piggyback';
+		if ( !empty( $target ) ) {
+			$redirectUrl .= sprintf( "?target=%s", $target );
+		}
+		$out->redirect( $redirectUrl );
```

Comment:
Indeed it does. Unauthorised users get redirected to sign in and upon signing in back to piggyback.